rerun_py/src/python_bridge.rs

```diff
@@ -276,6 +279,167 @@ fn flush_and_cleanup_orphaned_recordings(py: Python<'_>) {
     });
 }
 
+#[derive(FromPyObject)]
+enum DurationLike {
+    Int(i64),
+    Float(f64),
+    Duration(time::Duration),
+}
+
+impl DurationLike {
+    fn into_duration(self) -> time::Duration {
+        match self {
+            Self::Int(i) => time::Duration::from_secs(i as u64),
+            Self::Float(f) => time::Duration::from_secs_f64(f),
+            Self::Duration(d) => d,
+        }
+    }
+}
+
+/// Defines the different batching thresholds used within the RecordingStream.
+#[pyclass(name = "ChunkBatcherConfig")]
+#[derive(Clone)]
+pub struct PyChunkBatcherConfig(ChunkBatcherConfig);
+
+#[pymethods]
+impl PyChunkBatcherConfig {
+    #[new]
+    #[pyo3(
+        text_signature = "(self, flush_tick=None, flush_num_bytes=None, flush_num_rows=None, chunk_max_rows_if_unsorted=None)"
+    )]
+    /// Initialize the chunk batcher onfiguration.
+    ///
+    /// Parameters
+    /// ----------
+    /// flush_tick : int | float | timedelta | None
+    ///     Duration of the periodic tick, by default `None`.
+    ///     Equivalent to setting: `RERUN_FLUSH_TICK_SECS` environment variable.
+    ///
+    /// flush_num_bytes : int | None
+    ///     Flush if the accumulated payload has a size in bytes equal or greater than this, by default `None`.
+    ///     Equivalent to setting: `RERUN_FLUSH_NUM_BYTES` environment variable.
+    ///
+    /// flush_num_rows : int | None
+    ///     Flush if the accumulated payload has a number of rows equal or greater than this, by default `None`.
+    ///     Equivalent to setting: `RERUN_FLUSH_NUM_ROWS` environment variable.
+    ///
+    /// chunk_max_rows_if_unsorted : int | None
+    ///     Split a chunk if it contains >= rows than this threshold and one or more of its timelines are unsorted,
+    ///     by default `None`.
+    ///     Equivalent to setting: `RERUN_CHUNK_MAX_ROWS_IF_UNSORTED` environment variable.
+    fn new(
+        flush_tick: Option<DurationLike>,
+        flush_num_bytes: Option<u64>,
+        flush_num_rows: Option<u64>,
+        chunk_max_rows_if_unsorted: Option<u64>,
+    ) -> Self {
+        let default = ChunkBatcherConfig::DEFAULT;
```

Comment:
this should be `from_env`
Otherwise setting one of these parameters means that all the other environment variables are no longer honored!
This also has the advantage that a user can retrieve the environments' setting :)